src/main/scala/uk/gov/hmrc/initprototype/HerokuFormation.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.initprototype
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+case class HerokuFormation(size: String, quantity: Int, app: HerokuApp)
+
+object HerokuFormation {
```

Comment:
Do we need these objects and readers defined explicitly, given that we have the case classes and the JSON matches the case class structure in terms to property names and types?

Reply:
I wondered about that but then wondered if we should be explicit to define the mapping and not be tied to the Heroku API representation?

Reply:
Also do we want to use snake case rather than camel case in the case classes e.g. for HerokuRelease?